files/drafts/RFC-0001/RFC-0001.md

```diff
@@ -1,56 +1,84 @@
-author:
-- Author 1
-
-# Don't change the following lines
-header-includes:
-- \usepackage{fancyhdr}
-- \pagestyle{fancy}
-- \fancyfoot[L]{-release-version-}
-output: pdf_document
----
-
-# RFC-0001: RFC on Glossary
-
-This file is the template to be used for new RFCs. The first thing to do should always be the modification of the title, subtitle and initial author(s) in the `YAML` header block at the beginning of this file. The title and subtitle need to be in single quotes and all authors need to be on a separate line following the pattern `- author_name`.
-
-The first headline directly above this first paragraph should be changed to contain the RFC-id and the same title as the one being used in the header content.
-
-# Markdown
-
-It is allowed to use the whole range of Markdown features as well as everything supported by Pandoc out of the box. A good overview and introduction can be found in the Markdown specification itself[^daring_markdown] and the Pandoc user documentation[^pandoc_markdown].
-
-## Hyperlinks
-
-RFCs are released as PDF documents. While it is possible to set hyperlinks in Markdown that will also work in PDF documents it is advised to add the actual links in footnotes. This way they can be easily read.  Footnotes (similar to the one in the previous paragraph can be created by adding `[^footnote_name]` where the footnote is to be placed and `[^footnote_name]: Content of the footnote` at the end of this file. Only alphanumeric characters and underscores are allowed. Hyperlinks in footnotes should be added in angled brackets: `<http://example.com>`.
-
-## Images
-
-It is possible to include images. They should be added in an `images` folder directly next to the markdown document. In the text they are referenced like this: `![Image caption](./images/image_name_including_file_ending.png)`. The path is relative to the location of the Markdown file.
-
-## Diagrams
-
-In addition to pre-created images, it is also possible to add so-called *Mermaid*[^mermaid] diagrams and flowcharts to the document. This are two examples:
-
-```mermaid
-graph LR
-    draft[RFC Draft] --> candidate[RFC Candidate] --> draft
-    candidate --> release{RFC release}
-    release -.updates on existing RFCs.-> draft
-```
-
-```mermaid
-pie
-    title Time spent on RFCs
-    "Preparation" : 10
-    "Draft" : 60
-    "Release Candidate" : 30
-```
-
-<!-- Footnote area: Please keep the list of footnotes sorted alphabetically to simplify managing them -->
-
-[^daring_markdown]: <https://daringfireball.net/projects/markdown/basics>
-[^mermaid]: <https://mermaid-js.github.io/mermaid/#/>
-[^pandoc_markdown]: <https://pandoc.org/MANUAL.html#pandocs-markdown>
+- - - -
+title: 'RFC on RFC Glossary'
+subtitle: 'Time Machine RFC-0001'
+author:
+* Frederic Kaplan
+# Don't change the following lines
+header-includes:
+* \usepackage{fancyhdr}
+* \pagestyle{fancy}
+* \fancyfoot[L]{-release-version-}
+output: pdf_document
+- - - -
+# RFC-0001:  Time Machine Glossary
```

Comment:
Stylistically: there should be empty lines around headers (according to some common style preferences). This is not technically wrong but we should think about whether or not we want to also consider the format of changes submitted by contributors. I am thinking to include a style-checking step in the toolchain that automatically tests for consistent formatting but this makes it of course more difficult for contributors because they have to take care how to format their text.


Example for the preferred formatting:

```
# Heading 1

## Heading 1.1

Text
```